ttnn/cpp/ttnn/operations/eltwise/binary/binary.hpp

```diff
@@ -227,6 +249,22 @@ constexpr auto ne_ = ttnn::register_operation_with_auto_launch_op<
     "ttnn::ne_",
     operations::binary::InplaceRelationalBinary<operations::binary::BinaryOpType::NE>>();
 
+constexpr auto rsub_binary = ttnn::register_operation_with_auto_launch_op<
```

Comment:
Why do we add _binary suffix here?
The version with scalar is also pretty much binary. 
from my point of view these ops always accept two arguments and are binary. You can choose different device operations to use, but hide that behind a single op structure with multiple invoke overloads.

Reply:
@ayerofieiev-tt These api's _binary api's are not exposed in python side, or in composite ops. I am using them inside a single struct ExecuteRsub from `binary_composite` where one variant of the op is unary device operation and other is binary device operation. There are some functions in each device operation that are not exposed like `preprocess_inputs` etc. and inline `impl` functions, either I'll have to duplicate them in composite file or expose them in .hpp

Reply:
@ayerofieiev-tt you wanna skip this register_operation for `rsub_binary` and call the invoke of this struct instead ? I think I got it.

Reply:
The outcome I am looking for is next:

cpp
```cpp
ttnn::pow(tensor, scalar);
ttnn::pow(tensor, tensor);
```
and py
```py
ttnn.pow(tensor, scalar)
ttnn.pow(tensor, tensor)
```

instead of 
```cpp
ttnn::pow(tensor, scalar);
ttnn::pow_binary(tensor, tensor);
```

Reply:
I think rsub and other bitwise ops were moved from `unary` to `binary_composite`. 
ttnn::pow is still in `unary_composite` I see, this should go to `binary_composite` instead